scipy/stats/tests/test_continuous.py

```diff
@@ -1031,19 +1031,25 @@ def test_shapes(self):
 
 
 class TestMakeDistribution:
-    @pytest.mark.parametrize('i, distdata', enumerate(distcont))
+    @pytest.mark.parametrize('i, distdata', enumerate(distcont + distdiscrete))
     def test_make_distribution(self, i, distdata):
         distname = distdata[0]
 
         slow = {'argus', 'exponpow', 'exponweib', 'genexpon', 'gompertz', 'halfgennorm',
                 'johnsonsb', 'kappa4', 'ksone', 'kstwo', 'kstwobign', 'powerlognorm',
-                'powernorm', 'recipinvgauss', 'studentized_range', 'vonmises_line'}
+                'powernorm', 'recipinvgauss', 'studentized_range', 'vonmises_line',
+                'betanbinom', 'zipf', 'logser', 'skellam'}  # discrete
```

Comment:
`betabinom`, `zipf`, and `logser` are all slow because for the parameters tested, the median is at the left end of the support, so we run into the last shortcoming of the [top post](https://github.com/scipy/scipy/pull/22312#issue-2782644701).